### PR TITLE
Add logic for inferring admission_date

### PR DESCRIPTION
**Pull request by zacharylawrence**

### CODE REVIEW

File: recidiviz/persistence/converter/booking.py [hunk 1]
```diff
@@ -29,26 +31,37 @@ def copy_fields_to_builder(booking_builder, proto, metadata):
...
     release_date_inferred = None if release_date is None else False
 
     return release_date, release_date_inferred
+
+
+def _parse_admission(proto):
+    admission_date = fn(parse_date, 'admission_date', proto)
+
+    if admission_date is None:
+        admission_date = datetime.datetime.now()
```
Comment (terinpw) [inline recidiviz/persistence/converter/booking.py]: I think we want this to be the last_seen_time.

**Comment by terinpw** [comment]
Also now that we're inferring before entity matching, we can probably close #348. 

-- There's a small TODO in entity_matching.py related to it (need to update `if db_booking.admission_date_inferred:` to become `if db_booking.admission_date_inferred and ingested_booking.admission_date_inferred`.

File: recidiviz/persistence/converter/booking.py (hunk 1 above)
Comment (zacharylawrence) [reply recidiviz/persistence/converter/booking.py]: Good point, updated!

**Comment by zacharylawrence** [comment]
> Also now that we're inferring before entity matching, we can probably close #348.
> 
> -- There's a small TODO in entity_matching.py related to it (need to update `if db_booking.admission_date_inferred:` to become `if db_booking.admission_date_inferred and ingested_booking.admission_date_inferred`.

Good catch, done!

**Comment by zacharylawrence** [comment]
> You should be able to mark 348 closed?

Looks like it's already closed

**Pull request merged**